index.bs

```diff
@@ -2852,7 +2852,19 @@ Objects</a>.</p>
 
        <li><p>Interpret |value| as an integer, and let |number| be that number.</p></li>
 
-       <li><p>Let |region|'s <a>WebVTT region lines</a> be |number|.</p></li>
+       <li><p><i>Conversion</i>: Let <var>S</var> be the set of finite IEEE 754 double-precision
```

Comment:
It's likely we don't need this `conversion` label

Reply:
It may be useful if we need to do this conversion elsewhere as well. Might be worth checking.